docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-promo-block-external.html.twig

```diff
@@ -0,0 +1,59 @@
+{# ###### Setup Image Position & Image ####### #}
+{% set image_position = content.field_image_position_promo|render|striptags|lower|trim %}
+{% if image_position == 'no-img' %}
+    {% set image_position = '' %}
+{% elseif image_position == 'left' %}
+    {% set image_position = 'nci-promo-block--with-image' %}
+    {% set image = content.field_override_image_promotional %}
+{% else %}
+    {% set image_position = 'nci-promo-block--with-image nci-alternating-right' %}
+    {% set image = content.field_override_image_promotional %}
+{% endif %}
+
+{# ###### Setup Theme ####### #}
+{% set promo_theme = content.field_ncids_theme|render|striptags|lower|trim %}
```

Comment:
Basically same contents as everywhere else. Use the paragraph entity to get at these fields.